### ISSUE TITLE
Wrong calculation

### ISSUE THREAD

**Issue by philippecade** [philippecade/howlong#3]
![image](https://user-images.githubusercontent.com/804768/152346212-b09ce6fb-ff3e-4a06-94b1-2c3589aa0477.png)

Console output:
```
Time left 522
howlong.js:28 Found 791 work false time 0
howlong.js:29 	iterationCount 0
howlong.js:28 Found 820 work true time 29
howlong.js:29 	iterationCount 1
```

**Comment by philippecade** [comment]
Another example:
![image](https://user-images.githubusercontent.com/804768/152822890-3996665b-432f-4cab-8996-dd51712b81b7.png)
```
Time left 522
howlong.js:28 Found 451 work false time 0
howlong.js:29    iterationCount 0
howlong.js:28 Found 503 work true time 52
howlong.js:29    iterationCount 1
howlong.js:22 Time left 440
howlong.js:28 Found 754 work false time 52
howlong.js:29    iterationCount 2
```

**Comment by philippecade** [comment]
Fixed by #4.
